doc/test/hirte_e2e.md

```diff
@@ -0,0 +1,18 @@
+## Hirte testing e2e
+
+### Environment Setup
+
+E2E tests rely on containers as seprate compute entities,
+Use of containers enble to simulaute hirte functional behaviour
+on single compute.
+
+Test directory contains two container files
+- Containerfile-build
+- Containerfile
+
+Containerfile-build published in quay
+quay.io/yarboa/hirte-builder
```

Comment:
Maybe this can be done in a later step. But, I think we should move this image to the containers organization in quay: https://quay.io/organization/containers

Reply:
Ack, will need your help

Reply:
I don't have the permissions for this organization. @rhatdan can you help?

Reply:
I don't have access to that (I think)
@edsantiago @TomSweeneyRedHat PTAL

Reply:
@edsantiago sure. But, we don't have access to the `libpod` organization either. How should it be set up?

Reply:
We are not going to distribute hirte as a containers, it doesn't make sense. And here we are creating containers for each test run, so I don't see a reason why such containers should be published on quay.io. Or am I missing something?

Reply:
I will explain,
Currently the build workflow download the following:
https://github.com/containers/hirte/blob/main/.github/workflows/build.yml#L14-L29

Is running on css9 container, refer,  https://github.com/containers/hirte/blob/main/.github/workflows/build.yml#L13-L14
It install rpms, 
this container quay.io/libpod/hirte-builder or other name, replace L14-L29

Each time build run it pulls the build image and starts with meson commands

@ygalblum suggest to move this part into the build workflow, prepare test containers, and in e2e workflow consume hirte-images for e2e tests,

Does it make sense?

Reply:
The containers created each run are created for testing purposes and are not published at all. To reduce their build time, @Yarboa created a base container , `hirte-builder` (which may change, but only when needed), currently published in his private user account in Quay. 

Keeping containers images on someone's private account is a bad practice as it means that only they can push new images. For that I proposed that we find a better place to store this container. I suggested quay.io, but maybe a better place to put it would be here in GitHub along side the code.

Reply:
@ygalblum 
Issue created #140

Reply:
@pypingou @ygalblum 
Why can't we use AutoSD repo?

Reply:
Uploading to registry
registry.gitlab.com/centos/automotive/sample-images/hirte/hirte-builder:1.0.0

Reply:
Found temporary home, till  decision #140 is made